resources/rest-service/cloudify/migrations/versions/1fbd6bf39e84_4_5_to_4_5_5.py

```diff
@@ -50,7 +50,6 @@ def upgrade():
     op.add_column('executions',
                   sa.Column('scheduled_for', UTCDateTime(), nullable=True))
 
-    op.execute('COMMIT')
```

Comment:
Was this commit never required then?

Reply:
Can't see why would it be, other than to break the "single transaction" semantics and show a warning

Reply:
Fair enough- no additional comments, and I see @mateumann greened it, so merge away.